### ISSUE TITLE
 Issue with LangChain v0.0.335 - Error in ChatOpenAI Callbacks Expected Runnable Instances

### ISSUE THREAD

**Issue by girithodu** [langchain-ai/langchain#15779]
### Issue you'd like to raise.

I am currently utilizing LangChain version 0.0.335 in my Fast API Python application. In the main.py file, the following code snippet is implemented:

main.py 
 ```
        streaming_model = ChatOpenAI(
            model_name="gpt-4",
            temperature=0.1,
            openai_api_key=os.getenv("OPENAI_API_KEY2"),
        )

        non_streaming_model = ChatOpenAI(
            model_name="gpt-4",
            temperature=0.1,
            openai_api_key=os.getenv("OPENAI_API_KEY2"),
        )

        retriever = vector_store.as_retriever()

        sales_persona_prompt = PromptTemplate.from_template(SALES_PERSONA_PROMPT)
        condense_prompt = PromptTemplate.from_template(CONDENSE_PROMPT)


        chain = ConversationalRetrievalChain.from_llm(
            llm=streaming_model,
            retriever=retriever,
            condense_question_prompt=condense_prompt,
            condense_question_llm=non_streaming_model,
            combine_docs_chain_kwargs={"prompt": sales_persona_prompt},
            verbose=True,
        )
        return chain(
            {"question": sanitized_question, "chat_history": conversation_history}
        )
        
   except Exception as e:
       return {"error": str(e)} 
```
        
However, this implementation throws the following error: 

 ` "error": "2 validation errors for LLMChain\nllm\n  instance of Runnable expected (type=type_error.arbitrary_type; expected_arbitrary_type=Runnable)\nllm\n  instance of Runnable expected (type=type_error.arbitrary_type; expected_arbitrary_type=Runnable)"`
       
    Expected Behavior:
I expected the code to execute without errors. The issue seems to be related to the expected types for the llm parameter in the ConversationalRetrievalChain.from_llm method.

Request for Assistance:
I kindly request assistance in understanding and resolving this issue. Any insights, recommendations, or specific steps to address the error would be highly appreciated.

Thank you for your time and support.

**Comment by hwchase17** [comment]
I'm pretty sure what's happening is you are on an older version of LangChain but you are using the new imports. I'm guessing you are importing:

```
from langchain_community.chat_models import ChatOpenAI
```
(or possibly from `langchain_openai`)

Is that correct?

If so, you should want to either:

1. Go back to `from langchain.chat_models import ChatOpenAI`
2. Upgrade your LangChain version

The TLDR of why this is happening is that the old LangChain version is doing type checking, but this is from before `langchain-core` was a thing. So it is type checking that it is `langchain.runnables.Runnable` or something. But anything imported from `langchain-community` is AFTER the core split, so it's of type `langchain_core.runnables.Runnable` or the like

Can you let me know if this works

**Comment by girithodu** [comment]
Thank you, Updating langchain has fixed the issue!

**Comment by shawnesquivel** [comment]
Confirmed the issue was fixed for me as well from upgrading from` 0.0.339` to `0.1.9`
